tests/__init.test.js

```diff
@@ -19,13 +19,14 @@ navigator.getVRDisplays = function () {
   return Promise.resolve([mockVRDisplay]);
 };
 
-require('index');
+const AFRAME = require('index');
 var AScene = require('core/scene/a-scene').AScene;
 // Make sure WebGL context is not created since Travix CT runs headless.
 // Stubs below failed once in a while due to asynchronous tesst setup / teardown.
 AScene.prototype.setupRenderer = function () {};
 
 setup(function () {
+  window.AFRAME = AFRAME;
```

Comment:
This one gave me a hard time. I'm not sure how this worked with browserify with karma before, I think karma-browserify injected the `AFRAME` variable in each file but I'm not really sure. With webpack with karma `AFRAME` was an empty object in some tests, defining as a global variable in `setup` fixed it.